apps/stress-test/src/components/v8/StressApp.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { getTestParams } from '../../shared/testParams';
+import { performanceMeasure } from '../../shared/performanceMeasure';
+import { StressContainer } from './StressContainer';
```

Comment:
The pattern of creating and explicitly importing components to test isn't very flexible and can be a bit of work to maintain. The test case code is identical in all 3 stress apps.

You should perhaps consider creating the apps and test cases dynamically based on folder name and `require.context`. This way someone can add new stress components and new stress containers without needing to modify the underlying code that runs the tests. This is done in the griffel benchmark app 

https://github.com/microsoft/griffel/blob/main/apps/benchmark/src/implementations/index.ts

There just by adding a new implementation folder and components, the scenarios and tests are automatically generated and appear on the app without further work

Reply:
Good call! Made this change and things are _much_ nicer now.